### PR TITLE
Dark theme, Localstorage, Fun fact and minor refactoring

### LINKED ISSUE #5: Local storage support
Tasks added by the user should persist on refreshing or reopening the app

### PR DESCRIPTION
**Pull request by Tanay-J**
This small PR includes
- Dark theme
- Local storage support for task list and theme. User can now refresh or reopen the app and still can see their tasks and chosen theme. Fixes #5 
- Fun fact about Pomodoro technique at the bottom of the landing page
- Minor code refactoring

**Preview**
https://pomodoro-app-git-localstorage-plus-tanay-j.vercel.app/

### CODE REVIEW

**Review by junaidshaikh-js — APPROVED** [review]
Great work! 

Clean code! Love th use of random fun facts. Good to merge.

File: src/components/landing/landing.module.css [hunk 1]
```diff
@@ -5,3 +5,6 @@
 .btn {
   padding: 0.5rem 2.5rem;
 }
+.info_text{
+  /* max-width: 30rem; */
+}
```
Comment (JowelTisso) [inline src/components/landing/landing.module.css]: * Remove unused code in PR

File: src/components/navigation/Navbar.jsx [hunk 2]
```diff
@@ -1,13 +1,45 @@
-import { BsLightbulbFill } from "react-icons/bs";
+import { useEffect, useState } from "react";
+import styles from "./navigation.module.css";
+import { BsMoonFill, BsSunFill } from "react-icons/bs";
+import { Link } from "react-router-dom";
 
 const Navbar = () => {
+  const [darkMode, setDarkMode] = useState(
+    JSON.parse(localStorage.getItem("darkMode")) || false
+  );
```
Comment (JowelTisso) [inline src/components/navigation/Navbar.jsx]: * You can extract localStorage function separately

**Review by JowelTisso — APPROVED** [review]
Rest Looks Good

File: src/components/landing/landing.module.css (hunk 1 above)
Comment (Tanay-J) [reply src/components/landing/landing.module.css]: Thanks for pointing out. Removed.

File: src/components/navigation/Navbar.jsx (hunk 2 above)
Comment (Tanay-J) [reply src/components/navigation/Navbar.jsx]: Done

**Pull request merged**